config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -1,5 +1,42 @@
 # this file should contain all periodic jobs that use the fejta-bot token
 periodics:
+- name: periodic-api-review-help
+  interval: 1h
+  decorate: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      command:
+      - /commenter
+      args:
+      - |-
+        --query=org:kubernetes
+        is:pr
+        base:master
+        label:kind/api-change
+        -label:api-review
+        -label:do-not-merge/work-in-progress
```

Comment:
Why exclude needs-rebase, hold and wip?

Reply:
PRs with those labels aren't generally ready for API review